systers_portal/static/css/style.css

```diff
@@ -247,6 +251,18 @@ table.decoration-none  tr a {
   margin: 30px 0px;
 }
 
+.text-indent35 {
+  text-indent: 35px;
+}
+
+/* Meetup Location Members Grid CSS
+-------------------------------------------------- */
+.crop_img {
+  width: 100px;
+  height: 100px;
+  overflow: hidden;
+}
+
```

Comment:
Finally I found [django-imagekit](https://www.djangopackages.com/packages/p/django-imagekit/) app who can help us. This is the grid that comes out by using `django-imagekit `
http://gyazo.com/96ab933e661313bcc8f3b01304fa0065

So let me know if it looks good, I can push a patch that uses imagekit.